src/searches.py

```diff
@@ -32,14 +35,19 @@ class Searches:
     def __init__(self, browser: Browser, searches: RemainingSearches):
         self.browser = browser
         self.webdriver = browser.webdriver
-        # Share search terms across instances to get rid of duplicates
-        if Searches.searchTerms is None:
-            Searches.searchTerms = self.getGoogleTrends(
-                searches.desktop + searches.mobile
-            )
-            # Shuffle in case not only run of the day
-            random.shuffle(Searches.searchTerms)
-            # todo write shuffled searchTerms to disk to better emulate actual searches
+
+        self.googleTrendsShelf: shelve.Shelf = shelve.open("google_trends")

```

Comment:
add `/google_trends` to `.gitignore`

Reply:
Right, a few other files are generated by shelf as well. Waiting until tomorrow's run to add them all to `.gitignore`